forge-gui/res/cardsfolder/upcoming/mine_worker.txt

```diff
@@ -2,12 +2,11 @@ Name:Mine Worker
 ManaCost:2
 Types:Artifact Creature Assembly-Worker
 PT:2/1
-A:AB$ Branch | Cost$ T | BranchConditionSVar$ UrzaCheck | BranchConditionSVarCompare$ GE2 | FalseSubAbility$ GainLife | TrueSubAbility$ GainLifeUrza | SpellDescription$ You gain 1 life. If you control creatures named Power Plant Worker and Tower Worker, you gain 3 life instead.
-SVar:GainLife:DB$ GainLife | LifeAmount$ 1
-SVar:GainLifeUrza:DB$ GainLife | LifeAmount$ 3
-SVar:UrzaCheck:SVar$X/Plus.Y
-SVar:X:Count$Valid Creature.YouCtrl+namedTower Worker/LimitMax.1
-SVar:Y:Count$Valid Creature.YouCtrl+namedPower Plant Worker/LimitMax.1
+A:AB$ GainLife | Cost$ T | LifeAmount$ X | SpellDescription$ You gain 1 life. If you control creatures named Power Plant Worker and Tower Worker, you gain 3 life instead.
```

Comment:
I bet this needs a StackDescription

Reply:
![image](https://user-images.githubusercontent.com/67333662/201170795-44fe4322-ea9e-4a90-9660-3de362aac8bf.png)
Actually it correctly tells on the stack if you gain 1 or 3 life

Reply:
Unless you do a corner case:
- have 2 of the 3 workers in play
- activate the ability (stack will say you get 1 life)
- before that ability resolves on the stack cast the other worker with flash somehow
- now when that ability resolves, you will get 3 life

Reply:
Ah I see ! I guess StackDescription = SpellDescription ?